{{cookiecutter.github_repo}}/bin/build.sh

```diff
@@ -6,10 +6,10 @@ if [ $# == 0 ]; then usage; fi
 while getopts ":dp" opt; do
     case "$opt" in
         d)
-          docker-compose build --build-arg DEBUG=true api
+          sudo docker-compose build --build-arg DEBUG=true api
```

Comment:
These `sudo` additions seem out of place in this PR. What calls for them? Why put them in this pull request?

Reply:
That is (supposed to be) from a separate PR in this repo that Brian was fixing recently. My guess is that it mistakenly made it in here.

Reply:
correct, just merged in master, so going to close this as outdated